src/crimson/net/Socket.h

```diff
@@ -35,7 +35,7 @@ class Socket
     : sid{seastar::engine().cpu_id()},
       socket(std::move(_socket)),
       in(socket.input()),
-      out(socket.output()) {}
+      out(socket.output(65536)) {}
```

Comment:
magic number. could you add a very short comment to explain why we need to set it?

Reply:
Comment added.

Reply:
i failed to find the comment =(

Reply:
Sorry, seems I left them in another branch when doing tests, now they are included.